Picker/tests/Picker-specs.js

```diff
@@ -97,4 +97,18 @@ describe('Picker Specs', () => {
 		expect(actual).toHaveClass(expectedInline);
 		expect(actual).toHaveClass(expectedHeader);
 	});
+
+	test('should be able to receive custom voice control labels', () => {
```

Comment:
```suggestion
	test('should stringify custom voice control labels and set to \'data-webos-voice-labels-ext\'', () => {
```
Could you please check and fix it?